src/components/RadioInput/RadioInput.tsx

```diff
@@ -6,6 +6,7 @@ import Label from 'Label';
 type RadioInputProps = {
   className?: string;
   describedBy?: string;
+  disabled?: boolean;
```

Comment:
Could we use `isDisabled` as the prop name (to follow boolean prop name convention)